### ISSUE TITLE
Dataset geoshape:box documentation

### ISSUE THREAD

**Issue by valentinedwv** [ESIPFed/science-on-schema.org#236]
Can we make the suggestion that the doco for geoshape box, explicitly spell out that it is (lat lon, lat lon).  or (Y X, Y X)  for datasets using lat-lon. If it's not lat-lon specify the coordinate system.

https://github.com/ESIPFed/science-on-schema.org/blob/master/guides/Dataset.md#use-geoshape-for-all-other-location-types

**Comment by mbjones** [comment]
Yeah, I think it would be good to be more explicit about the ordering. We linked to the comment by @smrgeoinfo that describes the inconsistencies of point serialization by various vendors and apps: https://github.com/ESIPFed/science-on-schema.org/issues/101#issuecomment-720808142 Probably some of that should be lifted up and made part of the actual recommendation guideline document.

**Comment by mbjones** [comment]
Moved to 1.4 milestone as this wasn't completed in time for the 1.3.2 maintenance release.
